docs/USER_RECORD.md

```diff
@@ -863,6 +863,11 @@ on removable media the delay is selected very low to minimize the chance the
 home directory remains in unclean state if the storage device is removed from
 the system by the user).
 
+`accessMode` → The access mode currently in effect for the home directory iself.
+
+`fileSystemType` → The file system type backing the home directory: a short
+string, such as "btrfs", "ext4", "xfs".
```

Comment:
do we have an authoritative list we can point to beside these examples? or is this the one?

Reply:
So right now homed only supports these three file systems inside the LUKS volumes, hence I added support for them, but nothing else.

I figure sooner or later we should have a table somewhere which can translate the various file system magics into their string identifiers, using the same names as /proc/filesystems would use, building on the infra we have in src/basic/filesystems.[ch] already (and which already looks at the magic but doesn't include it in the generated tables). 

Once we have that we should drop this manual translation function and just use that.

I'll add an explicit TODO list item for this. It requires some work and I don't really think it fits into this PR.